conf/gulp-tasks/template/createprecompiletemplates.js

```diff
@@ -28,25 +32,40 @@ function createPrecompileTemplatesTask (locale, translator) {
  *
  * @param {Function} callback called when the task is finished
  * @param {string} locale
+ * @param {boolean} isSearchBarOnly If only the templates related to the SearchBar
+ *                                  should be included.
  * @param {Translator} translator
  * @returns {stream.Readable}
  */
-function _precompileTemplates (callback, locale, translator) {
-  const precompiledFileName = getPrecompiledFileName(locale);
+function _precompileTemplates (callback, locale, isSearchBarOnly, translator) {
+  const precompiledFileName = getPrecompiledFileName(locale, isSearchBarOnly);
   const processAST = ast => new TranslateHelperVisitor(translator).accept(ast);
-  return precompileTemplates(callback, precompiledFileName, processAST);
+
+  let templatesWrapper = './conf/templates/handlebarswrapper.txt';
+  let templatesStream = src('./src/ui/templates/**/*.hbs');
+  if (isSearchBarOnly) {
+    templatesWrapper = './conf/templates/handlebarswrapper-search-bar.txt';
```

Comment:
would we be able to modify handlebarswrapper.txt so that it could be shared? the wrapper.txt file is a lodash template passed into gulp-wrap, which I believe we can pass params into